README.md

```diff
@@ -41,7 +41,7 @@ Password: Password
 
 Create your env
 
-your connection string will be `mysql://admin@password@localhost:3306/db`
+your connection string will be `mysql://admin:password@localhost:3306/db`
```

Comment:
does admin work ? not user
#23 this issue related to this and he cleared that the user is `user`